todo/Controllers/AccountController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using todo.Models;
+using todo.Repositories;
+
+namespace todo.Controllers
+{
+    public class AccountController : ControllerBase
```

Comment:
```suggestion
    public class AccountsController : ControllerBase
```
tên controller trong .net là số nhiều nhé